### DISCUSSION TITLE
Chatglm3 的ceval评估结果很差！

### DISCUSSION THREAD

**Discussion by githubjqh** [zai-org/ChatGLM3#582] [Show and tell]
用ceval数据集和代码测试从modelscope下载的chatglm3-6b-base模型，最后精度结果平均值为30.65%。与https://cevalbenchmark.com/static/model.html?method=ChatGLM3-6B-base中给出的结果相差太远。谁知道什么原因吗？

**Comment by hsmygcs** [comment]
想问一下最后是怎么解决的呢？
